eng/common/scripts/Scan-files-against-links.ps1

```diff
@@ -0,0 +1,43 @@
+param (
```

Comment:
Lets call this script "get-markdown-files-from-changed-files.ps1. 

Call the different git diff commands directly in the script.